MyGet.ps1

```diff
@@ -0,0 +1,43 @@
+# Performs the build for MyGet and uploads the generated NuGet packages to
+# the "google-dotnet" feed.
+#
+# MyGet's build syncs the repo and subrepos, to build manually be sure to:
+# git submodule update --init ClientGenerator
+
+function PrintHeader($message) {
+    Write-Host "****************************************"
+    Write-Host $message
+    Write-Host "****************************************"
+}
+
+Write-Host ".NET Client Libraries, build shard $Env:BuildShard"
+
+# TODO(chrsmith): Document why we require VS v14 to build.
```

Comment:
Ack. I've updated the comment accordingly. Keeping as-is since it shouldn't (?) make a difference.